android/external/MakefileTemplate

```diff
@@ -367,23 +385,24 @@ clean-install:
 	-rm -rf $(LOCAL)
 
 ## Remove generated artifacts
-clean: $(EXTERNAL_SRCS_CLEAN) clean-assets clean-install libetpan-clean
+clean: $(EXTERNAL_SRCS_CLEAN) clean-assets clean-install botan-clean jitter-clean
 
 ## Remove generated artifacts and files
 clean-all: clean
 	-rm -rf *.git
 	-rm -rf *.stamp
 #------------------------------------------------------------------------------#
 
-.PHONY = clean clean-install clean-assets libetpan-clean \
+.PHONY = clean clean-install clean-assets \
 	libiconv-build libiconv-install \
 	libetpan-build \
 	uuid-build uuid-prebuild \
-	gmp-build  gmp-install \
-	botan-build  botan-install \
+	gmp-build gmp-install \
+	botan-build botan-install jitter-build \
+	botan-clean jitter-clean \
 	sequoia-deps-build \
-	openssl-build  openssl-install \
-	nettle-build  nettle-install \
+	openssl-build openssl-install \
+	nettle-build nettle-install \
```

Comment:
This is just the phony declaration, but there shouldn't be any actual openssl, nettle or gmp targets anywhere in the core part.

Reply:
re openssl:
https://github.com/plancksecurity/foundation-planckJNIWrapper/blob/develop/android/external/MakefileTemplate#L312
https://github.com/plancksecurity/foundation-planckJNIWrapper/blob/develop/android/external/MakefileTemplate#L175

Reply:
I removed nettle apparently with no issues, will try to do same with openssl.

Reply:
Openssl also removed.

What about gmp?

Reply:
gmp removed as well.